### DISCUSSION TITLE
Code/functions/classes related documentation

### DISCUSSION THREAD

**Discussion by RajathSVasisth** [google/filament#4985] [Q&A] [answered]
Is there a code/functions/method related documentation for android? The core documentation discusses about the design part but I wanted to just implement stuff.

**Comment by GokulDas027** [comment, marked as answer]
You can access the javadoc for the project, In android studio press `Ctrl+Q` on any Filament Api, and click on the `x on localhost` link. It's like filament android api reference documentation.

![image](https://user-images.githubusercontent.com/22936485/148174497-c8dd522a-c749-48ab-aa2c-2e5d531ba7d7.png)

**Reply by RajathSVasisth** [reply]
I found out later about the doxygen generated documentation but this new way also helps. Thanks
